docs/docs/core/helper-plugin/hooks/use-filter.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: useFilter
+description: API reference for the useFilter hook in Strapi
+tags:
+  - hooks
+  - helper-plugin
+  - i18n
+---
+
+Provides localized string search functionality that is useful for filtering or matching items in a list. Options can be
+provided to adjust the sensitivity to case, diacritics, and other parameters.
+
+Stolen from [`react-aria`](https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/i18n/src/useFilter.ts).
```

Comment:
Nit: Since these are public-facing docs: should we say "borrow" instead? 🙈

Reply:
Sure, if it makes you feel better lol